backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/imagegallery/BpkImageGallery.kt

```diff
@@ -20,20 +20,22 @@ package net.skyscanner.backpack.compose.imagegallery
 
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import net.skyscanner.backpack.compose.imagegallery.internal.BpkImageGalleryGridModal
 import net.skyscanner.backpack.compose.imagegallery.internal.BpkImageGallerySlideshow
 import net.skyscanner.backpack.compose.modal.BpkModal
 import net.skyscanner.backpack.compose.modal.BpkModalState
 import net.skyscanner.backpack.compose.modal.rememberBpkModalState
 import net.skyscanner.backpack.compose.navigationbar.NavIcon
 
 @Composable
-fun BpkImageGallery(
+fun BpkImageGallerySlideshow(
```

Comment:
renamed this as at least the grid one needs to have unique names as otherwise there'll be api clashes. this will make it clearer what happens.
none if this was released yet, so not a breaking change